src/openforms/emails/digest.py

```diff
@@ -97,11 +100,13 @@ class InvalidCertificate:
 
     @property
     def admin_link(self) -> str:
-        form_admin_url = reverse(
-            "admin:simple_certmanager_certificate_change",
-            kwargs={"object_id": self.id},
+        form_relative_admin_url = furl(
+            reverse(
+                "admin:simple_certmanager_certificate_change",
+                kwargs={"object_id": self.id},
+            )
         )
-        return form_admin_url
+        return build_absolute_uri(form_relative_admin_url.url)
```

Comment:
```suggestion
        return build_absolute_uri(reverse("admin:simple_certmanager_certificate_change", kwargs={"object_id": self.id}))
```

No need for `furl` here